core/domain/story_domain.py

```diff
@@ -1083,6 +1089,13 @@ def update_thumbnail_filename(self, thumbnail_filename):
                 story.
         """
         self.thumbnail_filename = thumbnail_filename
+        file_system_class = fs_services.get_entity_file_system_class()
+        fs = fs_domain.AbstractFileSystem(file_system_class(
+            feconf.ENTITY_TYPE_STORY, self.id))
+        filename_prefix = 'thumbnail'
```

Comment:
Is the filepath constructed elsewhere in the codebase (e.g. when updating/saving the thumbnail)? If so, how do they handle the filename prefix and should we extract the construction of the filepath into a common method?

Reply:
The method used in saving a thumbnail is [here](https://github.com/oppia/oppia/blob/9b0478905fc8037c23536546af7b6819a805d498/core/controllers/editor.py#L673). We tried to use a similar approach here. WDYT ?

Reply:
Extract the common part and put it in a common constant.

Reply:
added in constants and wrote a test for same. Waiting for CI to see if the test passing or not. 
Tried running the test locally but it didn't run. 

```
----------------------------------------
Tasks still running:
  constants_test (started 18:13:43)
----------------------------------------
Error 1


------------------------------------------------------
    WARNING: FAILED TO RUN constants_test

    This is most likely due to an import error.
------------------------------------------------------

Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/Users/akshaynandwana/Desktop/opensource/oppia/scripts/run_backend_tests.py", line 512, in <module>
    main()
  File "/Users/akshaynandwana/Desktop/opensource/oppia/scripts/run_backend_tests.py", line 472, in main
    raise Exception('WARNING: No tests were run.')
Exception: WARNING: No tests were run.
```

Reply:
I tried to remove the test I added and also getting the same error.

Reply:
I will try this today after meeting - https://github.com/oppia/oppia/issues/10064#issuecomment-823798955

Reply:
this didn't work.

Reply:
<img width="839" alt="Screenshot 2021-07-03 at 23 25 33" src="https://user-images.githubusercontent.com/25057618/124363071-b82a2500-dc56-11eb-925e-b7c242c3a547.png">

Now, I am able to run backend tests locally.